.github/workflows/ci.yml

```
@@ -29,6 +29,10 @@ jobs:
         if: startsWith(matrix.rust, 'nightly')
         run: cargo check -Z features=dev_dep
       - run: cargo test
+      - run: cargo test --test loom --features loom
```

```suggestion
      - run: cargo test --test loom
```
`--features loom` adds `cfg(feature="loom")`, which isn't used anywhere and not defined in `Cargo.toml`. `cfg(loom)` is used, but unrelated to that.

---

Some people [tend to complain](https://github.com/smol-rs/concurrent-queue/pull/27#discussion_r973840359) when `loom` shows up in their `Cargo.lock` file, even if it never builds. By making `loom` optional, we can keep it out of the lockfile.